README.md

```diff
@@ -475,22 +487,51 @@ NOTE: There is another command: `test-watch-debug`, it is the same with `test-wa
 
 Can also use Postman to test the API in a more real environment. Import the API spec from [here](https://github.com/Greenstand/treetracker-wallet-api/blob/master/docs/api/spec/treetracker-wallet-api-v1-10.yaml).
 
-To run a local server with some seed data, run command:
+1. To set up a local server with some seed data, temporarily copy the following (from serverTest.js) and paste into server/server.js:
```

Comment:
can you add a new script and remove the serverTest.js file altogether? So to set up a local server with some seed data would still be npm run server-test but the server-test script would be `npm run test-seed and npm run server`